docs/cli.rst

```diff
@@ -98,6 +98,11 @@ If that is the entire fragment name, a random hash will be added for you::
    Whether to start ``$EDITOR`` to edit the news fragment right away.
    Default: ``$EDITOR`` will be started unless you also provided content.
 
+.. option:: --section SECTION
+
+   The section to use for the news fragment.
+   Default: the first section with no path.
```

Comment:
I don't fully understand the sections... but can you have *multiple sections* without a path ?

Reply:
It wouldn't make sense and wouldn't work.
It should be considered invalid to have multiple sections with the same path (even the same path of `""`). No need to add complexity to this ticket, but the config warning of that might be worthy of a separate issue.

Reply:
Opened #606

Reply:
Agree that we can have this on #606 

So, maybe we should not add any comment about the default behaviour here.